packages/conf-rust-llvm/conf-rust-llvm.1/opam

```diff
@@ -17,7 +17,7 @@ depexts: [
   ["rust-llvm"] { os-family = "alpine" }
   ["rust-llvm"] { os-family = "arch" }
   ["rust-llvm"] { os-family = "debian" }
```

Comment:
```suggestion
  # ["rust-llvm"] { os-family = "debian" } # see below
```

Reply:
This should be reverted.

Reply:
Debian was failing with package not existent and succeeds with this change (since rust-lld is now installed by rustc there as well). Maybe it depends on the version?